access-protected-api/README.md

```diff
@@ -0,0 +1,162 @@
+  "tokenEncryptionKeyId": null,
+  "defaultRedirectUri": null,
+  "certification": null,
+  "optionalClaims": null,
+  …
+}
+```
+
+<!-- TODO: Link to first tutorial in series when published. -->
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- Java 8+
+- Maven
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Configure an application to expose a web API](https://docs.microsoft.com/azure/active-directory/develop/quickstart-configure-app-expose-web-apis) to register the sample API.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting | Value for this sample app                                            | Notes                                                                                       |
+|:------------------------------:|:---------------------------------------------------------------------|:--------------------------------------------------------------------------------------------|
+| **Name**                       | `java-api`                                                           | Suggested value for this sample. <br/> You can change the app name at any time.             |
+| **Supported account types**    | **Accounts in this organizational directory only (Single tenant)**   | Suggested value for this sample.                                                            |
+| **Platform type**              | _None_                                                               | No redirect URI required; don't select a platform.                                          |
+| **Client secret**              | _**Value** of the client secret (not its ID)_                        | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it). |
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Update code sample with app registration values
+
+Open the [_application.yml_](src/main/resources/application.yml) file and modify the four Azure Active Directory configuration properties using the values from your [app's registration in the Azure portal](https://docs.microsoft.com/azure/active-directory/develop/quickstart-configure-app-expose-web-apis).
+
+```yaml
+# 'Tenant ID' of your Azure AD instance - this value is a GUID
+tenant-id: value-here
+
+# 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+client-id: value-here
+
+# Client secret 'Value' (not its ID) from 'Client secrets' in app registration in Azure portal
+client-secret: value-here
+
+# 'Object ID' of app registration in Azure portal - this value is a GUID
+client-object-id: value-here
+```
+
+### 3. Install package(s) and compile application
+
+To install Spring Boot and MSAL libraries:

```

Comment:
```suggestion
To install Spring Boot and the MSAL for Java ([msal4j](https://mvnrepository.com/artifact/com.microsoft.azure/msal4j)) packages:
```

Reply:
Oh, but it's not this one.  It's actually a Spring Boot meta package which includes msal4j as a downstream dependency.  Does that change thoughts on this one?

Reply:
I'll go ahead and merge this one as-is, but if you want to noodle on this a bit more for the other Spring READMEs, feel free to. :)  Have a great weekend.

Reply:
Yup, will need to figure that out. Thanks, you too!